src/fpump.jl

```diff
@@ -390,16 +385,17 @@ function fpump(optimizer, m)
                 real_status,real_sol, real_obj = generate_real_nlp(optimizer, m, mip_sol; random_start=true)
                 cnlpinf += 1
             end
-            if state_is_optimal(real_status) || (real_status == MOI.ALMOST_LOCALLY_SOLVED && m.options.allow_almost_solved_integral)
-                if real_status == MOI.ALMOST_LOCALLY_SOLVED
-                    @warn "Integral feasible point only almost locally solved. Disable with `allow_almost_solved_integral=false`"
-                    only_almost_solved = true
+            if state_is_optimal(real_status) || (only_almost_solved(real_status) && m.options.allow_almost_solved_integral)
+                if only_almost_solved(real_status)
+                    @warn "Integral feasible point only almost solved. Disable with `allow_almost_solved_integral=false`"
                 end 
                 nlp_obj = real_obj
                 nlp_sol = real_sol
                 iscorrect = true
                 break
             elseif are_type_correct(nlp_sol, m.var_type, m.disc2var_idx, catol)
+                # TODO: maybe there is a different better status code
```

Comment:
@ccoffrin don't know what the best status is in this situation. 
Here the feasibility pump was able to find an integral solution but optimizing it (changing the real valued variables) wasn't successful. In that case the values from the nlp are just plugged into the objective function to get an objective value.

Reply:
I would say `LOCALLY_SOLVED`/`FEASIBLE_POINT` is the status there.  Just make sure to throw a warning.

Reply:
Does this depend on the solver? Is it possible to call it OPTIMAL if the nl solver is globally optimal?
Warning is thrown ;)

Reply:
Nothing that comes out of the feasibility pump can be called `OPTIMAL` regardless of the solver.

Reply:
haha thanks that's a very valid point totally forgot that this is all in the feasibility pump.